RNPurchases.podspec

```diff
@@ -16,14 +16,14 @@ Pod::Spec.new do |spec|
   spec.source_files = "ios/**/*.{h,m,swift}"
   spec.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
 
-  # Ignore the Purchases.framework that would get downloaded by the download script, meant for 
+  # Ignore the Purchases.framework that would get downloaded by the download script, meant for
```

Comment:
we should probably remove this at some point, I'm honestly not sure whether manual install works anymore